src/main/java/seedu/address/model/pet/DateOfBirth.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.pet;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Pet's date of birth.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDateOfBirth(String)}
+ */
+public class DateOfBirth {
+
+    public static final String MESSAGE_CONSTRAINTS = "Date of Birth must follow the format of yyyy-mm-dd.";
+    public static final String DATE_OF_BIRTH_FORMAT = "yyyy-mm-dd";
+
+
+    public final String value;
```

Comment:
maybe store it as a `java.time.LocalDate` instead of simply String?

Reply:
Updated `value` as `java.time.LocalDate`